Xamarin.Forms.Platform.Android/CollectionView/ItemsViewAdapter.cs

```diff
@@ -16,23 +16,20 @@ namespace Xamarin.Forms.Platform.Android
 	public class ItemsViewAdapter : RecyclerView.Adapter
 	{
 		protected readonly ItemsView ItemsView;
-		readonly Context _context;
```

Comment:
This was removed, but do we know why we added it in the first place?

Reply:
It used to be necessary for setting the `Platform` property on each item as they were created. Now that `Platform` is gone, it's no longer necessary.